### PR TITLE
add another example to ch-run

### PR DESCRIPTION
**Pull request by MFTabriz**
Thanks for all your helps @terminalforlife. Please check this one too. ameowattention

### CODE REVIEW

File: sheets/ch-run [hunk 1]
```diff
@@ -5,3 +5,9 @@
 # Run the given `echo` command inside a Charliecloud container, using the
 # unpacked image at '/data/foo'.
 ch-run /data/foo -- echo hello
+
+# Make the Charliecloud container writable (-w) and mount the host's
```
Comment (terminalforlife) [inline sheets/ch-run:9]: I'd probably opt for `-w` in the parentheses.

**Comment by terminalforlife** [comment]
Aside from that one thing, I'm liking what I'm seeing. The reason I opt for quotes around things like paths, is to avoid any confusion as to when the path begins and ends, plus a lot of us are probably used to seeing quotes around paths and filenames from commands like `ls`. I tend to use graves (`) when referring to actual code, such as flags. That's just what I personally use as a guideline. The main thing I shoot for, is consistency; I'm just as anal in my code!

I think one of the hardest things about this sort of project, is consistency, due to how different people post stuff with few if any stylistic or coding guidelines set in place. Why does this matter? Because it might be really confusing to people using this project. Although I'd say, in this regard, this project has come a long way from when I first saw it.

That being said, seeing different styles is useful, too. I suppose I hadn't considered that ages ago, when I started committing changes to the contrary. For example, if one person commits an example wherein you have squished flags (Ex: `-abc`), and another person commits an example with a more traditional flag layout (Ex: `-a -b -c`), it might then confuse cheat.sh readers and maybe even other committers, but it also demonstrates two different methods by which to use these single-letter flags.

Sorry, I went a bit off-topic there. lol

**Comment by MFTabriz** [comment]
Thanks again for your helps. I totally agree with you on the consistency. I believe having a more verbose style guide can be really helpful.
> 
> Aside from that one thing, I'm liking what I'm seeing. The reason I opt for quotes around things like paths, is to avoid any confusion as to when the path begins and ends, plus a lot of us are probably used to seeing quotes around paths and filenames from commands like `ls`. I tend to use graves (`) when referring to actual code, such as flags. In Markdown and when working with traditional shells, the grave is known for its use in command substitution, so most of us are probably familiar with it 'holding code', so to speak
> 
> That's just what I personally use as a guideline. The main thing I shoot for, is consistency; I'm just as anal in my code!
> 
> I think one of the hardest things about this sort of project, is consistency, due to how different people post stuff with few if any stylistic or coding guidelines set in place. Why does this matter? Because it might be really confusing to people using this project. Although I'd say, in this regard, this project has come a long way from when I first saw it.
> 
> That being said, seeing different styles is useful, too. I suppose I hadn't considered that ages ago, when I started committing changes to the contrary. For example, if one person commits an example wherein you have squished flags (Ex: `-abc`), and another person commits an example with a more traditional flag layout (Ex: `-a -b -c`), it might then confuse cheat.sh readers and maybe even other committers, but it also demonstrates two different methods by which to use these single-letter flags.
> 
> Sorry, I went a bit off-topic there. lol

**Comment by chubin** [comment]
@terminalforlife See it the same, especially this part: 

> different styles is useful, too. 

Using cheat sheets like this is not only a good opportunity to learn more
about some command, but also a good opportunity to learn some style

**Comment by terminalforlife** [comment]
Maybe I should be less aggressive in my commits, then, regarding styles. Perhaps it's better the user be confused and encouraged to learn *why* they're confused, than to coddle them into submission of the traditional or by-the-book way. This is inspiring me to make a YT video. Lol

**Pull request merged**